docs/core/tracer.md

```diff
@@ -170,42 +162,48 @@ def get_s3_object(bucket_name, object_key):
 
 ## Asynchronous and generator functions
 
-<Note type="warning">
-  <strong>We do not support async Lambda handler</strong> - Lambda handler itself must be synchronous
-</Note><br/>
+!!! warning
+    **We do not support async Lambda handler** - Lambda handler itself must be synchronous
 
 You can trace asynchronous functions and generator functions (including context managers) using `capture_method`.
```

Comment:
The change was proposed by @heitorlessa , any thoughts on that?

Reply:
I'll look into it this week more carefully.

The content tab suggestion was to save reader screen, so one can look at what's relevant instead of every option at once.

If I understood Tom's note is around the code split lacking the Lambda handler made it more confusing, and instead have a long block with all options (as it was)

Reply:
We could still split it into 3 separate blocks if we want to save screen space, but in that case we should copy the lambda handler code into each block. In that case, the lambda handler code should only show it calling the relevant function (either async, generator or context mgr rather than all 3).